linux_os/guide/system/software/integrity/crypto/var_system_crypto_policy.var

```diff
@@ -0,0 +1,17 @@
+documentation_complete: true
+
+title: 'The system-provided crypto policies'
+
+description: |-
+    Specify the crypto policy (string) for the system.
+
+type: string
+
+operator: equals
+
+interactive: false
+
+options:
+    default: DEFAULT
+    legacy: LEGACY
+    future: FUTURE
```

Comment:
There is FIPS policy as well. Not sure if it is already released, but it will be there sooner or later:
https://gitlab.com/redhat-crypto/fedora-crypto-policies/blob/master/back-ends/profiles/FIPS.pl

Reply:
It is there.

Reply:
Added `FIPS` and `NEXT`